tool/README.md

```diff
@@ -33,7 +33,27 @@ git checkout -b release_$(date +%s);
 
 #### Update the DevTools version number
 
+##### Releasing a clean version
```

Comment:
"clean" isn't clear to me - can you be specific what you mean here? do you mean non-dev release?